### PR TITLE
update ember-cli and ember-source to 3.4 LTS

### PR DESCRIPTION
**Pull request by 0xadada**
* [x] Updated ember-cli
* [x] Updated ember-source
* [x] Updated ancillary packages

related to PR https://github.com/poteto/ember-deep-set/pull/15#discussion_r224120203

### CODE REVIEW

**Comment by snewcomer** [comment]
Thanks @0xadada!  Merged.  Rebase master and this should be g2g!

**Comment by 0xadada** [comment]
@snewcomer all set.

File: package.json [hunk 1]
```diff
@@ -24,25 +24,26 @@
     "ember-cli-babel": "^7.1.2"
   },
   "devDependencies": {
-    "broccoli-asset-rev": "^2.4.5",
-    "ember-ajax": "^3.0.0",
-    "ember-cli": "~2.17.1",
-    "ember-cli-dependency-checker": "^2.0.0",
+    "broccoli-asset-rev": "^3.0.0",
+    "ember-ajax": "^3.1.2",
```
Comment (snewcomer) [inline package.json]: Can we remove this?

**Review by snewcomer — COMMENTED** [review]
Real quick - Mind updating the ember-changeset PR?  Also one minor comment.  Otherwise thanks a ton!

https://github.com/poteto/ember-changeset/pull/320/files

**Comment by 0xadada** [comment]
re https://github.com/poteto/ember-deep-set/pull/17#pullrequestreview-163814478
@snewcomer done, and done

**Pull request merged**